README.rst

```diff
@@ -0,0 +1,96 @@
+========
+Overview
+========
+
+.. start-badges
+
+.. list-table::
+    :stub-columns: 1
+
+    * - docs
+      - |docs| |readthedocs|
+    * - build status
+      - |testing| |codestyle| |coverage|
+
+.. |docs| image:: https://github.com/swxsoc/swxsoc/actions/workflows/docs.yml/badge.svg
+    :target: https://github.com/swxsoc/swxsoc/actions/workflows/docs.yml
+    :alt: Documentation Build Status
+
+.. |testing| image:: https://github.com/swxsoc/swxsoc/actions/workflows/testing.yml/badge.svg
+    :target: https://github.com/swxsoc/swxsoc/actions/workflows/testing.yml
+    :alt: Build Status
+
+.. |codestyle| image:: https://github.com/swxsoc/swxsoc/actions/workflows/codestyle.yml/badge.svg
+    :target: https://github.com/swxsoc/swxsoc/actions/workflows/codestyle.yml
+    :alt: Codestyle and linting using flake8
+
+.. |coverage| image:: https://codecov.io/gh/swxsoc/swxsoc/branch/main/graph/badge.svg?token=C6JLIWZRBU 
+    :target: https://codecov.io/gh/swxsoc/swxsoc
+    :alt: Testing coverage
+
+.. |readthedocs| image:: https://readthedocs.org/projects/swxsoc/badge/?version=latest
```

Comment:
I'd need help setting up a new `readthedocs` site.

Reply:
I think i set it up at https://readthedocs.org/projects/swxsoc/ i can add you as an admin if you want.